zookeeper/src/test/java/com/linecorp/armeria/client/zookeeper/TestBase.java

```diff
@@ -49,7 +49,9 @@
     @BeforeClass
     public static void start() {
         try {
-            zkInstance = ZKFactory.apply().withRootDir(ROOT_FOLDER.newFolder("zookeeper")).create();
+            if (zkInstance == null) {
```

Comment:
@anuraaga Thank you. You are right, i forgot to close the connection in the test case setting up stage, also leave some comments in your PR.